nav2_amcl/src/amcl_node.cpp

```diff
@@ -649,17 +649,16 @@ AmclNode::requestMap()
 
   auto request = std::make_shared<nav2_tasks::MapServiceClient::MapServiceRequest>();
 
-  RCLCPP_INFO(get_logger(), "AmclNode::requestMap: invoking service call");
+  RCLCPP_INFO(get_logger(), "AmclNode: Processing request map service.");
```

Comment:
We should remove all of the node names in the message strings that are redundant with the name returned from get_loger. For example, I think this message would output "[amcl] AmclNode: Processing request map service." Instead, we can just have "[amcl] Processing request map service." That is, let the RCLCPP_INFO/DEBUG/etc. macro output the node name and not include it in the message. Also, there is another issue that relates to this topic: #59

Reply:
Is that the case? I thought you only get the "[amcl]" if you used a named logger: get_logger("amcl")

Reply:
Yeah, when invoking get_logger() from a node, it'll use the node's name in the output string in brackets. When invoke from some other (non-node) context, you have to use rclcpp::get_logger("your string"). In either case though, you'll have the name (node name or your provided string) in brackets.

Reply:
Ah ok. I think what my fix would be here is to change the get_logger() --> get_logger("AmclNode"), I know that it will be overwritten as your say, but I think being explicit is good in most contexts to make sure there's no room for misinterpretation.

Reply:
@SteveMacenski I don't think putting is the name in the get_logger call is the trick. It's basically:

1. Node created with its name: Node("amcl")
2. Logging throughout the Node's code with: RCLCPP_X(get_logger(), "Message to output");

This will result in an output like: "[amcl]: Message to output"

Reply:
@mjeronimo the problem with that appropriate is that not everything is a node (i.e. static costmap layer, plugins/optional pointers) that are distinctively different programs and should be logged separately. I would not want to have `[costmap_2d]` appear for a static layer program as then the error wouldnt be meaningful in context to what's happening in a large, complex node. This is just one particular example to highlight.